html/answers/dhanush-q4.html

```diff
@@ -0,0 +1,71 @@
+<html>
+    <head>
+
+        <!-- Global site tag (gtag.js) - Google Analytics -->
+        <script async src="https://www.googletagmanager.com/gtag/js?id=UA-157837769-1"></script>
+        <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+        gtag('config', 'UA-157837769-1');
+        </script>
+        
+        <title>
+            HTML | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content="Explain the importance of the HTML DOCTYPE?">
+    </head>
+    <body>
+		
+		<img src="../../assets/CN_logo.png" height="72px" alt="Coding Ninjas Logo" />
+
+        <div>
+            <p>
+                <small>
+
+                    <a href="../../index.html">Home</a>
+                    &gt; <a href="../index.html">HTML Questions</a>
+                    &gt; Explain the importance of the HTML DOCTYPE?
+
+                </small>
+               
+            </p>
+        </div>
+
+
+        <content>
+            <div>
+                <h4>
+                    <h4>Explain the importance of the HTML DOCTYPE?</h4>
+                </h4>
+            </div>
+            <br>
+            <br>
+            <article>
+                <strong>Answer:</strong> 
+                <p>
+                    <ul>
+                        <li>DOCTYPE is an instruction to the web browser regarding what version of the markup language the page is written. </li>
```

Comment:
We got an almost same question earlier, add a different one.

Reply:
Changes done

Reply:
Resolved Q4 conflict as I changed my questio no to 5